keps/sig-multicluster/1645-multi-cluster-services-api/README.md

```diff
@@ -708,12 +714,16 @@ this cluster.
 
 #### ClusterSetIP
 
-A non-headless `ServiceImport` is expected to have an associated IP address, the
-clusterset IP, which may be accessed from within an importing cluster. This IP
-may be a single IP used clusterset-wide or assigned on a per-cluster basis, but
-is expected to be consistent for the life of a `ServiceImport` from the
-perspective of the importing cluster. Requests to this IP from within a cluster
-will route to backends for the aggregated Service.
+A non-headless `ServiceImport` is expected to have associated IP addresses, the
+clusterset IPs, which may be accessed from within an importing cluster. These IPs
+may be used clusterset-wide or assigned on a per-cluster basis, but is expected
+to be consistent for the life of a `ServiceImport` from the perspective of the
+importing cluster. Requests to these IPs from within a cluster will route to
+backends for the aggregated Service. The `IPs` field must correspond to the
+protocol defined in the `ipFamilies` field if defined. How the `ipFamilies`
+field is set is implementation defined, for instance it might correspond to what
+IP protocol the local cluster support or globally reconciled like other
+ServiceImport fields.
```

Comment:
Sorry, a couple minor adjustments that I missed before.

```suggestion
backends for the aggregated Service. The `IPs` field must correspond to the
protocols defined in the `ipFamilies` field, if specified. How the `ipFamilies`
field is determined is implementation-defined, for instance it might correspond to what
IP protocols the exporting clusters support or globally reconciled like other
ServiceImport fields.
```

Reply:
No worries, thanks again for the suggestion!

Reply:
My intent with referring to the local cluster is about some possible implementation that have some kind of gateway that didn't really need to care about IP protocol and the globally reconciled was actually the "globally reconciled part". So I ended up changing the last sentence vs what you were proposing but let me know if you have further suggestion/something better that I wrote after your comment.